virtual-coach-server issue 108: create database dump for testing

raar1 opened the pull request:
Fixes PerfectFit-project/virtual-coach-server#108

This PR adds tools for dumping and restoring the perfectfit db, and a `test.dump` file containing sample user data for use in development and testing.

svenvanderburg commented on README.md:
```diff
@@ -47,7 +47,28 @@ postgres db. For example, you can print out the database metadata:
 
-# Use the python package
+# Backup and restore the db
+Important: The database docker-compose must be up and running (as described above) before running any of the following steps.
```
Maybe also mention that migrations need to be applied too.

raar1 replied:
Agreed and done.

svenvanderburg reviewed (approved):
Nice, this is superhandy :)

You can make a few improvements:
* Move scripts to a `script` folder (update readme accordingly)
* Add a few lines in the readme on adding a new user (or other type of data) to the database. Do you use SQL for that or python/sqlalchemy? I think the latter should be safer and easier to use.

raar1 commented:
I've moved the scripts to a `utils/` directory and updated the README. I also made the dump and restore scripts handle non-running db cases gracefully.

OK I've added a README section about adding users, with a link to the onboarding/ dir in the other repo. That directory contains a README explaining how to use it.

All interactions with the db in our backend code (Rasa actions, onboarding etc) is done with python and sqlalchemy. There should be no raw sql anywhere.

> Or maybe use the onboariding script?
The problem with this is that it requires the niceday-api to be up and running, so it's not convenient if you want to write tests (since it will be fetching real data from the remote senseserver). So a `test.dump` with dummy data in it is more convenient for dev work and tests.
